src/main/kotlin/eu/europa/ec/eudi/sdjwt/vc/SdJwtVcVerifier.kt

```diff
@@ -228,7 +262,7 @@ private suspend fun issuerJwkSource(
  */
 internal sealed interface SdJwtVcIssuerPublicKeySource {
 
-    data class Metadata(val iss: Url, val kid: String?) : SdJwtVcIssuerPublicKeySource
+    data class Metadata(val iss: Url) : SdJwtVcIssuerPublicKeySource
```

Comment:
Why was  `kid` removed?

Just double-checking this.

I think that we don't need it, since Nimbus would take it into consideration with its processor.

Reply:
SD-JWT-VC doesn't say anything about `kid` during Issuer Metadata resolution.